recipe/meta.yaml

```diff
@@ -7,12 +7,13 @@ package:
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/affinegap-{{ version }}.tar.gz
-  sha256: 02faa7579df8d98beafd40bb924b7a3a9d4e42edf6938e297366903054e4ef61
+  # Use upstream archive tarball for tests
+  url: https://github.com/dedupeio/affinegap/archive/refs/tags/v1.12.tar.gz
+  sha256: 2bf275b6d8b083bdb32b16fd4e4a55025eaad2ef6f325d3f0b0cdf8253608aaf
 
 build:
   number: 2
```

Comment:
```suggestion
  number: 0
```

Reply:
I did wonder about this.  Our feedstock had previously been forked (by ProServ?) from conda-forge with a `build_number` of `2` but not added to defaults.

So, rather than complicate matters with them, I left it as is (and nearly asked a question on Slack 😆 ).

Reply:
It doesn't matter so much here. The CFFL forks were a hackday project to auto build tons of noarch packages (if I recall). If you look at conda-forge, their latest build number is 3, incremented for a new python version.

Reply:
OK, so

- ignore the CFFL work
- reset `build_number` to `0` if it is our first build